### PR TITLE
Description of Payloadmask

### PR DESCRIPTION
**Pull request by ChrSchultz**
Created  Description of Payloadmask in Readme.md, 
renamed the original paxcounter.conf and platformio.ini to paxcounter_orig.conf and platform_orig.ini and updated Readme.md 
added paxcounter.conf and platformio.ini to .gitignore

### CODE REVIEW

File: src/paxcounter_orig.conf [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 // Set this to include BLE counting and vendor filter functions, or to switch off WIFI counting
 #define VENDORFILTER                    0       // set to 0 if you want to scan all devices, not filtering smartphone OUIs
-#define BLECOUNTER                      0       // set to 0 if you do not want to install the BLE sniffer
+#define BLECOUNTER                      1       // set to 0 if you do not want to install the BLE sniffer
```
Comment (cyberman54) [inline src/paxcounter_orig.conf:19]: BLEcounter is intentional default off, because it has massive impact on wifi sniffer.

File: src/paxcounter_orig.conf [hunk 2]
```diff
@@ -25,11 +25,11 @@
 #define BLESCANINTERVAL                 80      // [illiseconds] scan interval, see below, 3 .. 10240, default 80ms = 100% duty cycle
 
 // Corona Exposure Notification Service(ENS) counter
-#define COUNT_ENS                       0       // count found number of devices which advertise Exposure Notification Service
+#define COUNT_ENS                      1       // count found number of devices which advertise Exposure Notification Service
                                                 // set to 0 if you do not want to enable this function
 
 // for additional sensors (added by some user)
-#define HAS_SENSOR_1                    0       // set to 1 if you want to transmit CWA counter
```
Comment (cyberman54) [inline src/paxcounter_orig.conf:32]: Do we want this as default? [suggestion: no]

File: src/loarconf_sample_h [hunk 3]
```diff
@@ -1,89 +1,88 @@
-#ifndef __LORACONF_H__
```
Comment (cyberman54) [inline src/loarconf_sample_h:1]: Why did you change this filename? [suggestion: keep it]

File: platformio_orig.ini [hunk 4]
```diff
@@ -132,4 +132,4 @@ upload_protocol = esptool
 upload_protocol = esptool
 build_type = debug
 platform = https://github.com/platformio/platform-espressif32.git#develop
-platform_packages = framework-arduinoespressif32 @ https://github.com/espressif/arduino-esp32.git
+platform_packages = framework-arduinoespressif32 @ https://github.com/espressif/arduino-esp32.git
```
Comment (cyberman54) [inline platformio_orig.ini:135]: please remove this edit, it changes nothing (?)

File: README.md [hunk 5]
```diff
@@ -481,6 +481,15 @@ Send for example `8386` as Downlink on Port 2 to get battery status and time/dat
 0x14 set payload mask
 
 	byte 1 = sensor data payload mask (0..255, meaning of bits see above)
```
Comment (cyberman54) [inline README.md:483]: should now read "...meaning of bits see below"

File: README.md [hunk 6]
```diff
@@ -85,11 +85,11 @@ By default bluetooth sniffing not installed (#define *BLECOUNTER* 0 in paxcounte
 
 Compile time configuration is spread across several files. Before compiling the code, edit or create the following files:
 
-## platformio.ini
-Edit `platformio.ini` and select desired hardware target in section boards. To add a new board, create an appropriate hardware abstraction layer file in hal subdirectory, and add a pointer to this file in sections boards.
+## platformio_orig.ini
+Edit `platformio_orig.ini` and select desired hardware target in section boards. To add a new board, create an appropriate hardware abstraction layer file in hal subdirectory, and add a pointer to this file in sections boards. and rename to `platformio.ini`
```
Comment (cyberman54) [inline README.md:89]: should read "...board. Rename file to `platformio.ini`"

**Review by cyberman54 — CHANGES_REQUESTED** [review]
Thanks for this PR! I am going to merge it, but added some remarks. Could you please check them?

File: src/paxcounter_orig.conf (hunk 2 above)
Comment (ChrSchultz) [reply src/paxcounter_orig.conf:32]: remade

File: src/loarconf_sample_h (hunk 3 above)
Comment (ChrSchultz) [reply src/loarconf_sample_h:1]: checked, file is correct I didn't change it...

    #ifndef __LORACONF_H__
    #define __LORACONF_H__

File: src/paxcounter_orig.conf (hunk 1 above)
Comment (ChrSchultz) [reply src/paxcounter_orig.conf]: done

File: platformio_orig.ini (hunk 4 above)
Comment (ChrSchultz) [reply platformio_orig.ini]: nothing, is correct checked.

**Comment by ChrSchultz** [comment]
done changes, ready for merging

**Comment by cyberman54** [comment]
what about loarconf_sample_h, maybe this is  accidentially in the commit?

**Comment by ChrSchultz** [comment]
loraconf_sample.h is correct, I copied the original into the directory and renamed to loraconf_sample.h and git-diff wrote the wrong text accidently...

File: src/loarconf_sample_h (hunk 3 above)
Comment (ChrSchultz) [reply src/loarconf_sample_h]: ok now renamed to loraconf_sample.h

**Comment by cyberman54** [comment]
Could you, please, remove from the PR all your commits regarding loraconf.sample.h?
You made no changes to the contents of this file, thus it should kept untouched by the PR.
After this is cleaned, i will merge the PR.

**Comment by ChrSchultz** [comment]
I tried to delete loraconf_sample.h from PR, don't know weather it works...

**Comment by cyberman54** [comment]
Merged into development for further inspection.
